Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
+
+>a. Agent Pools - Read & Manage
+>b. Deployment Groups - Read & Manage
+
+3. Create a service principle and add it to your subscription as Owner. This is the subscription that will be used to host the image and Agent VM.
+
+4. Add the service principle to the subscription that will host the image and agent VMs with Owner role.
+
+## Build a Packer Image
+
+1. Build a Docker image and a container from the Docker image by mounting `Agent` folder as a volume in your Docker container.
+
+```
+docker run -it -v <path-to-vdc-folder>/Agent:/usr/app/src/Agent vdc:latest
+```
+
+2. Update the run.sh file with the service principle and subscription information.
+
+3. Run the command below from PowerShell terminal to build an VM Image
+
+```
+bash -c "./build.sh"
+```
+
+## Create a VM from Packer Image
+
+1. Run the command below from PowerShell terminal to create an instance from the VM Image
+
+```
+bash -c "./vsts-agent-create.sh <organization-url> <personal-access-token> <vsts-pool-name> <agent-vm-name>"
```

Comment:
This step create a VM resource in Azure, not technically an instance.

Add more information here, where are you creating the VM? You can add information that the VM will be created in the Subscription and Tenant specified in step number ...

Reply:
Updated.